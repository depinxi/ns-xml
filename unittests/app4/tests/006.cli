-c abc def ghi --invalid-switch sub "This should succeed (even invalid switch will break the multiarg value parsing)"