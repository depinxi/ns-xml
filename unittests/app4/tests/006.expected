CLI: "-c", "abc", "def", "ghi", "--invalid-switch", "sub", "This should succeed (even invalid switch will break the multiarg value parsing)"
Value count: 1
Values: "This should succeed (even invalid switch will break the multiarg value parsing)"
Error count: 1
Subcommand: sub
switchA=False
argumentB=
multiargC=abc def ghi
sub_argumentD=
sub_switchE=False
